### DISCUSSION TITLE
Add new field type for collection under "Presentation"

### DISCUSSION THREAD

**Discussion by kent-leow** [directus/directus#14327] [Feature Requests]
An image presentation field needed. Or even a video.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
